Source/Core/VideoCommon/VertexLoaderManager.cpp

```diff
@@ -392,7 +392,7 @@ int RunVertices(int vtx_attr_group, OpcodeDecoder::Primitive primitive, int coun
 
     count = loader->RunVertices(src, dst.GetPointer(), count);
 
-    if (can_cpu_cull && !cullall)
+    if (can_cpu_cull && !cullall) [[unlikely]]
```

Comment:
This feels like a bad place to use `[[unlikely]]` - CPU culling is a user-specifiable option (Graphics -> Advanced -> Misc -> Cull Vertices on the GPU), see [the progress report](https://dolphin-emu.org/blog/2023/02/12/dolphin-progress-report-december-2022-january-2023/#50-18355-allow-dolphin-to-manually-cull-vertices-on-the-cpu-by-tellowkrinkle). It doesn't improve performance in Super Mario Galaxy, but it does help in other games.

Reply:
My reasoning here was that it's off by default and only really useful for a very small number of games. But I'd be perfectly okay with dropping it ofc.